src_fun/parser.cpp

```diff
@@ -538,8 +580,42 @@ static bool parse_expression_leaf(Token_Stream* stream, Block_Builder* builder,
                 return false;
 
             Token_Info_String* info = (Token_Info_String*) get_token_info(stream->ctx, name);
+            if (!info->value)
+                return ReportError(stream->ctx, name, "Expected a non-empty string literal as the import path."_s);
+
+            String trimmed = trim(info->value);
+            if (trimmed.length < info->value.length)
+                return Report(stream->ctx)
+                    .intro(SEVERITY_ERROR, name)
+                    .message("Import paths can not start nor end with whitespace."_s)
+                    .done();
+

```

Comment:
Why? You can name files with spaces at the front or end. I mean clearly stupid but no need for this limitation.

Reply:
This would make sense if it was a mistake, and the compiler checked that the file with spaces doesn't exist, but without spaces does.